packages/wrangler/src/__tests__/configuration.test.ts

```diff
@@ -547,18 +552,69 @@ describe("normalizeAndValidateConfig()", () => {
 				expect(config).toEqual(expect.objectContaining(expectedConfig));
 				expect(diagnostics.hasWarnings()).toBe(true);
 				expect(diagnostics.renderWarnings()).toMatchInlineSnapshot(`
-			          "Processing wrangler configuration:
-			            - \\"assets\\" fields are experimental and may change or break at any time."
-		        `);
+					"Processing wrangler configuration:
+					  - [1mDeprecation[0m: \\"legacy_assets\\":
+					    The existing behavior of the experimental \`assets\` feature will be changing on August 15th.
```

Comment:
If they have actually used `legacy_assets` in their config then the probably should not get this warning.

Reply:
I was unsure if we want to also warn users that legacy-assets will also probably go away soon - checking with Tanushree on that and the exact copy, otherwise will probably remove that :)